README.md

```diff
@@ -1,41 +1,8 @@
-<a name="readme-top"></a>
-
-<!--
-!!! IMPORTANT !!!
-This README is an example of how you could professionally present your codebase. 
-Writing documentation is a crucial part of your work as a professional software developer and cannot be ignored. 
-
-You should modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
-
-For more information on the importance of a professional README for your repositories: https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/readme_best_practices.md
--->
-
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] OPTIONAL: In this message:

![image](https://github.com/Elerqsousy/grandeur-frontend/assets/57421823/7a4cfb00-bea2-4c7e-af62-50bfbf34850d)

Did you mean "Stay tuned"?

Because "tooned" is the name of a cartoon:


![image](https://github.com/Elerqsousy/grandeur-frontend/assets/57421823/9a922d2a-74d3-4971-b04b-2288ea21fb03)

And what we say in English to ask people to come back later:

![image](https://github.com/Elerqsousy/grandeur-frontend/assets/57421823/b853d421-b85d-43a2-9a30-e7a96d916300)


I suggest you correct what seems to be a spelling mistake. 👍